src/layman/layer/micka/csw.py

```diff
@@ -147,9 +148,13 @@ def get_template_path_and_values(workspace, layername, http_method=None):
         spatial_resolution = {
             'scale_denominator': scale_denominator,
         }
-    else:
+    elif file_type == settings.FILE_TYPE_RASTER:
         languages = []
-        spatial_resolution = None
+        spatial_resolution = {
+            'ground_sample_distance': None,
+        }
+    else:
+        raise NotImplementedError()
```

Comment:
Let's add more info about the problem. Something like "Unknown file type. File_type={file_type}".